RUFAS/graph_generator.py

```diff
@@ -236,14 +234,14 @@ def _validate_graph_filter(
                 )
         return graph_filter_validation_logs
 
-    def _prepare_plot_data(
+    def _log_non_int_float_data(
```

Comment:
```suggestion
    def _log_non_numerical_data(
```
This might be a cleaner name.